### PR TITLE
Cleaned up integration-cli/docker_api_create_test.go

### PR DESCRIPTION
**Pull request by ZJaffee**
As Part of "Proposal: use of checkers on Integration test #16756"

### CODE REVIEW

File: integration-cli/docker_api_create_test.go [hunk 1]
```diff
@@ -28,11 +27,9 @@ func (s *DockerSuite) TestApiCreateWithNotExistImage(c *check.C) {
...
 	status, resp, err = sockRequest("POST", "/containers/create?name="+name, config2)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusNotFound)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusNotFound)
 	expected = "No such image: test456:latest"
-	if !strings.Contains(string(resp), expected) {
-		c.Fatalf("expected: %s, got: %s", expected, string(resp))
-	}
+	c.Assert(!strings.Contains(string(resp), expected), checker.Equals,true,Commentf("expected: %s, got: %s", expected, string(resp)))
```
Comment (cpuguy83) [inline integration-cli/docker_api_create_test.go]: `c.Assert(string(resp), checker.Not(checker.Contains), expected)`

File: integration-cli/docker_api_create_test.go [hunk 2]
```diff
@@ -15,12 +16,10 @@ func (s *DockerSuite) TestApiCreateWithNotExistImage(c *check.C) {
...
 	status, resp, err := sockRequest("POST", "/containers/create?name="+name, config)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusNotFound)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusNotFound)
 	expected := "No such image: test456:v1"
-	if !strings.Contains(string(resp), expected) {
-		c.Fatalf("expected: %s, got: %s", expected, string(resp))
-	}
+	c.Assert(!strings.Contains(string(resp), expected), checker.Equals,true,Commentf("expected: %s, got: %s", expected, string(resp)))
```
Comment (cpuguy83) [inline integration-cli/docker_api_create_test.go]: c.Assert(string(resp), checker.Not(checker.Contains), expected)

**Comment by duglin** [comment]
@ZJaffee any chance of you combining these PRs into bigger ones.  One PR per func is far more tedious to review than one PR per file or even group of funcs.

**Comment by ZJaffee** [comment]
I can start having all this be one PR, but I am doing as one PR per File, which represents a single work item for the ticket #16756, where currently the way files are tracked is by there being a numbered PR tied  to each item.

**Comment by vdemeester** [comment]
@ZJaffee @HuKeping and I tries to keep track on #16756 by putting the author and PR related to the files, but you can do a PR with multiple files no problems :wink: (there is no rule as one file one PR).

File: integration-cli/docker_api_create_test.go [hunk 3]
```diff
@@ -15,12 +16,10 @@ func (s *DockerSuite) TestApiCreateWithNotExistImage(c *check.C) {
...
 	status, resp, err := sockRequest("POST", "/containers/create?name="+name, config)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusNotFound)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusNotFound)
 	expected := "No such image: test456:v1"
-	if !strings.Contains(string(resp), expected) {
-		c.Fatalf("expected: %s, got: %s", expected, string(resp))
-	}
+	c.Assert(string(resp), checker.Not(checker.Contains), expected, Commentf("expected: %s, got: %s", expected, string(resp)))
```
Comment (vdemeester) [inline integration-cli/docker_api_create_test.go]: Here and everywhere else on this file : s/Commentf/check.Commentf/

Comment (vdemeester) [inline integration-cli/docker_api_create_test.go]: Also the comment is not needed here I feel.

File: integration-cli/docker_api_create_test.go [hunk 4]
```diff
@@ -28,11 +27,9 @@ func (s *DockerSuite) TestApiCreateWithNotExistImage(c *check.C) {
...
 	status, resp, err = sockRequest("POST", "/containers/create?name="+name, config2)
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusNotFound)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusNotFound)
 	expected = "No such image: test456:latest"
-	if !strings.Contains(string(resp), expected) {
-		c.Fatalf("expected: %s, got: %s", expected, string(resp))
-	}
+	c.Assert(string(resp), checker.Not(checker.Contains), expected, Commentf("expected: %s, got: %s", expected, string(resp)))
```
Comment (vdemeester) [inline integration-cli/docker_api_create_test.go]: Same here actually.

**Comment by HuKeping** [comment]
@vdemeester correct.

**Comment by jfrazelle** [comment]
@ZJaffee can you please update these

**Comment by calavera** [comment]
I'm very sorry but I'm going to close this PR until the errors are resolved. Please, feel free to send it again after testing that every test still works and the code is valid. `make test` and `make validate` are your friends.

As a rule of thumb, before opening all your PRs again, it will be easier to get them merged if you focus in one at a time, that way reviewers don't need to point the same errors over several changes.

**Pull request closed without merging**